### PR TITLE
Moar cleaning

### PR DESCRIPTION
**Pull request by GawainLynch**
* Added a tiny bug fix for the upload controller

### CODE REVIEW

File: src/Application.php [hunk 1]
```diff
@@ -268,10 +268,7 @@ public function initProfiler()
 
     public function initLocale()
     {
-        $configLocale = $this['config']->get('general/locale', Application::DEFAULT_LOCALE);
-        if (!is_array($configLocale)) {
-            $configLocale = [$configLocale];
-        }
+        $configLocale = (array) $this['config']->get('general/locale', Application::DEFAULT_LOCALE);
```
Comment (bobdenotter) [inline src/Application.php]: Does this work always? i did not know that..

Comment (GawainLynch) [inline src/Application.php]: `'foo'` will be cast to `['foo']`
`['foo']` will stay `['foo']`
`1` will be cast to `[1]`
`true` will be cast to `[true]`
…and so on

When I benchmarked it for 10,000,000 iterations the difference in performance was significant. :grin:

Comment (bobdenotter) [inline src/Application.php]: I use this a lot.. Faster, and three lines less of code. Awesome. :-)

**Comment by bobdenotter** [comment]
Merging in!

**Pull request merged**